shared/search/user-pane/index.js

```diff
@@ -49,7 +49,7 @@ export default connector.connect(
       return {
         mode: 'external',
         nonUserInfoProps: {
-          avatar: userForInfoPane.serviceAvatar,
+          avatar: userForInfoPane.extraInfo.serviceAvatar || '',
```

Comment:
so this is actually working. if its external we show the external avatar which is in the extra info